src/patternfly/components/Dropdown/styles.scss

```diff
@@ -0,0 +1,158 @@
+@import "../../patternfly-utilities";
+
+// Toggle
+$pf-c-dropdown__toggle--PaddingTop: $pf-global--spacer--xs !default;
+$pf-c-dropdown__toggle--PaddingRight: $pf-global--spacer--md !default;
+$pf-c-dropdown__toggle--PaddingBottom: $pf-global--spacer--xs !default;
+$pf-c-dropdown__toggle--PaddingLeft: $pf-global--spacer--sm !default;
+$pf-c-dropdown__toggle--FontSize: $pf-global--FontSize--md !default;
+$pf-c-dropdown__toggle--BackgroundColor: $pf-global--BackgroundColor--light-100 !default;
+$pf-c-dropdown__toggle--BorderWidth: $pf-global--BorderWidth--sm !default;
+$pf-c-dropdown__toggle--BorderColor: $pf-global--Color--dark-200 !default;
+$pf-c-dropdown__toggle--Color: $pf-global--Color--dark-100 !default;
+
+$pf-c-dropdown__toggle--hover--BorderWidth: $pf-global--BorderWidth--md !default;
+
+$pf-c-dropdown__toggle--is-expanded--BorderColor: $pf-global--Color--active !default;
+$pf-c-dropdown__toggle--is-expanded--BorderWidth: $pf-global--BorderWidth--md !default;
+
+// Toggle Arrow
+$pf-c-dropdown__arrow--MarginLeft: $pf-global--spacer--md !default;
+$pf-c-dropdown__arrow--BorderWidth: $pf-global--BorderWidth--md !default;
+
+// Menu
+$pf-c-dropdown__menu--BackgroundColor: $pf-global--BackgroundColor--light-200 !default;
+$pf-c-dropdown__menu--BoxShadow: $pf-global--BoxShadow--sm !default;
+
+// Menu Item
+$pf-c-dropdown__menu-item--PaddingTop: $pf-global--spacer--sm !default;
+$pf-c-dropdown__menu-item--PaddingRight: $pf-global--spacer--md !default;
+$pf-c-dropdown__menu-item--PaddingBottom: $pf-global--spacer--sm !default;
+$pf-c-dropdown__menu-item--PaddingLeft: $pf-global--spacer--md !default;
+$pf-c-dropdown__menu-item--first--PaddingTop: $pf-global--spacer--md !default;
+$pf-c-dropdown__menu-item--last--PaddingBottom: $pf-global--spacer--md !default;
+$pf-c-dropdown__menu-item--Color: $pf-global--Color--dark-100 !default;
+$pf-c-dropdown__menu-item--is-disabled--Color: $pf-global--Color--dark-200 !default;
+
+$pf-c-dropdown__menu-item--hover--BackgroundColor: $pf-global--BackgroundColor--light-300 !default;
+
+// Menu Item Separator
+$pf-c-dropdown__separator--Height: $pf-global--BorderWidth--md !default;
+$pf-c-dropdown__separator--BackgroundColor: $pf-global--BackgroundColor--light-300 !default;
+
+
+.pf-c-dropdown {
+  position: relative;
+  display: inline-block;
+
+  &__toggle {
+    padding: $pf-c-dropdown__toggle--PaddingTop $pf-c-dropdown__toggle--PaddingRight $pf-c-dropdown__toggle--PaddingBottom $pf-c-dropdown__toggle--PaddingLeft;
+    font-size: $pf-c-dropdown__toggle--FontSize;
+    color: $pf-c-dropdown__toggle--Color;
+    background-color: $pf-c-dropdown__toggle--BackgroundColor;
+    border: $pf-c-dropdown__toggle--BorderWidth solid $pf-c-dropdown__toggle--BorderColor;
+
+    &:hover {
```

Comment:
Button also uses `.pf-is-hover` and `.pf-is-active`. What's the convention to follow?

Reply:
@andresgalante or @srambach Do you have thoughts on this one? My understanding of these classes in the Button component was so that we can display all the variations of the button without actually having to hover or click on them.

Also, instead of `.pf-is-active` I use `.pf-is-expanded` which for the Dropdown are one in the same, but "expanded" seems more explicit about the state.

Reply:
@jgiardino is the expectation that on mouse down the menu is expanded? If not, then `active` is a different state. The bootstrap dropdown menu doesn't open on mouse down and has an `active` state just like a normal button. Actually at this point I'm a bit confused as to why we wouldn't simply utilize a the PF button as the toggle button for dropdown thereby allowing for all the different variants.

Reply:
I'm not sure I follow what you mean—when I look at the [bootstrap dropdown](https://getbootstrap.com/docs/4.1/components/dropdowns/#single-button) example, the menu is visible when I click on the toggle. And the styles applied when the toggle button is `:active` are the same styles applied when the menu is visible. I agree, they're two different things. I think what I meant was that the styles we would apply to the toggle in the active state should also get applied when the menu is expanded. But if that is true, then should I also add a `.pf-c-dropdown__toggle:active` selector to the styles currently defined for the `.pf-is-expanded .pf-c-dropdown__toggle` selector (i.e two comma-separated selectors)?

Reply:
The bootstrap dropdown button can be `:active` without being expanded. It occurs on mouse down. You can mouse down on the button (active state), then move the mouse away before you mouse up which won't opened the menu. True that once the dropdown is opened, the button has the same active treatment.